components/org.wso2.carbon.identity.api.server.configs/org.wso2.carbon.identity.api.server.configs.v1/src/main/java/org/wso2/carbon/identity/api/server/configs/v1/core/ServerConfigManagementService.java

```diff
@@ -381,6 +381,22 @@ public void patchCORSConfig(List<CORSPatch> corsPatchList) {
         }
     }
 
+    /**
+     * Get Home Realm Identifiers.
+     *
+     * @return List of home realm identifiers.
+     */
+    public List<String> getHomeRealmIdentifiers() {
+
+        IdentityProvider residentIdP = getResidentIdP();
+        String homeRealmIdStr = residentIdP.getHomeRealmId();
+        List<String> homeRealmIdentifiers = new ArrayList<>();
+        if (StringUtils.isNotBlank(homeRealmIdStr)) {
+            homeRealmIdentifiers = Arrays.stream(homeRealmIdStr.split(",")).collect(Collectors.toList());
```

Comment:
Do we need to trim the split strings?

Reply:
Addressed in https://github.com/wso2/identity-api-server/pull/260/commits/475f70efbd193afc7762fa8658b98f7cfedf290c